Assets/MRTK/SDK/Editor/Inspectors/Utilities/MeshOutline/BaseMeshOutlineInspector.cs

```diff
@@ -0,0 +1,157 @@
+    public class BaseMeshOutlineInspector : UnityEditor.Editor
+    {
+        private BaseMeshOutline instance;
+        private SerializedProperty m_Script;
+        private SerializedProperty outlineMaterial;
+
+        private string MrtkStandardShaderName => StandardShaderUtility.MrtkStandardShaderName;
+
+        private readonly Dictionary<string, object> defaultOutlineMaterialSettings = new Dictionary<string, object>()
+        {
+            { "_Mode", 5f },
+            { "_CustomMode", 0f },
+            { "_ZWrite", 0f },
+            { "_VertexExtrusion", 1f },
+            { "_VertexExtrusionValue",  0.01f },
+            { "_VertexExtrusionSmoothNormals", 1f },
+            { "_VERTEX_EXTRUSION", true },
+            { "_VERTEX_EXTRUSION_SMOOTH_NORMALS", true },
+        };
+
+        private void OnEnable()
+        {
+            instance = target as BaseMeshOutline;
+            m_Script = serializedObject.FindProperty("m_Script");
+            outlineMaterial = serializedObject.FindProperty(nameof(outlineMaterial));
+        }
+
+        public override void OnInspectorGUI()
+        {
+            DrawReadonlyPropertyField(m_Script);
+
+            EditorGUI.BeginChangeCheck();
+            EditorGUILayout.PropertyField(outlineMaterial);
+
+            var currentMat = instance.OutlineMaterial;
+            if (currentMat == null)
+            {
+                EditorGUILayout.HelpBox($"Outline Material field is empty, please create or select material", MessageType.Warning);
+
+                if (GUILayout.Button("Create && set new material"))
+                {
+                    outlineMaterial.objectReferenceValue = CreateNewMaterial();
+                }
+            }
+            else if(!IsCorrectMaterial(currentMat))
+            {
+                EditorGUILayout.HelpBox($"Material may not be configured correctly, check or reset to default", MessageType.Info);
+
+                if (GUILayout.Button("Update material settings to default"))
+                {
+                    ForceUpdateToDefaultOutlineMaterial(ref currentMat);
+                }
+            }
+
+            // Draw other properties
+            DrawPropertiesExcluding(serializedObject, nameof(m_Script), nameof(outlineMaterial));
+
+            if (EditorGUI.EndChangeCheck())
+                serializedObject.ApplyModifiedProperties();
+        }
+
+        private Material CreateNewMaterial()
+        {
+            var material = new Material(Shader.Find(MrtkStandardShaderName));
+            ForceUpdateToDefaultOutlineMaterial(ref material);
+            AssetDatabase.CreateAsset(material, $"Assets/{Selection.activeGameObject.name}Mat.mat");
+            return material;
+        }
+
+        private void ForceUpdateToDefaultOutlineMaterial(ref Material material)
+        {
+            if (!IsCorrectShader(material))
+            {
+                material.shader = Shader.Find(MrtkStandardShaderName);
```

Comment:
nit: You could use [this ](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/70d5731d959388a199143008d1c8ce7c4c4c577d/Assets/MRTK/Core/Utilities/StandardShader/StandardShaderUtility.cs#L21)in lieu of Shader.Find.

Reply:
Nice!